2024/5xxx/CVE-2024-5232.json

```
@@ -160,7 +160,7 @@
         "affected": [
           {
             "cpes": [
-              "cpe:2.3:a: campcodes:Complete Web Based School Management System:1.0:*:*:*:*:*:*:*"
+              "cpe:2.3:a:campcodes:Complete_Web_Based_School_Management_System:1.0:*:*:*:*:*:*:*"
```

Better in lowercase maybe

```suggestion
              "cpe:2.3:a:campcodes:Complete_Web_Based_School_Management_System:1.0:*:*:*:*:*:*:*"
```
```suggestion
              "cpe:2.3:a:campcodes:complete_web_based_school_management_system:1.0:*:*:*:*:*:*:*"
```